components/ui/Button.jsx

```diff
@@ -0,0 +1,8 @@
+export default function Button({ text, styles }) {
+  return (
+    <button
+      className={`font-Neufile  text-2xl font-normal not-italic leading-7  text-dark-grey  ${styles} `}>
```

Comment:
The feature missing on this className is the Hover effect since it's going to be reused everywhere the button will be called. kindly add that.

Reply:
Applied